pkg/commands/compute/init.go

```diff
@@ -121,13 +123,20 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	}
 	c.dir = dst
 
-	name, desc, authors, err := promptOrReturn(c.manifest, c.dir, c.Globals.File.User.Email, in, out)
+	// Assign the default profile email if available.
```

Comment:
There may well be no profiles created if it's a new CLI user and they go straight to wanting to use the C@E features. 

The 'email' variable here is actually just used as the 'authors' value in the fastly.toml and isn't required to be filled in. 

The user is always prompted to enter an author value, but if we're able to find the email from their token then we display that as the 'default' value, in case they enter nothing at the prompt that asks for the package author. Otherwise if there's no profile data to get an email from the is no default displayed and it just means no value is assigned to the author fields in the fastly.toml file.